locust/runners.py

```diff
@@ -352,6 +352,7 @@ def start_shape(self):
         self.update_state(STATE_INIT)
         self.shape_greenlet = self.greenlet.spawn(self.shape_worker)
         self.shape_greenlet.link_exception(greenlet_exception_handler)
+        self.environment.shape_class.start_time = monotonic()
```

Comment:
Could you not just call `self.environment.shape_class.reset_time()` here and that would be enough?